v22.1/take-full-and-incremental-backups.md

```diff
@@ -38,9 +41,10 @@ Collection:
 |—— incrementals
 ~~~
 
-[`SHOW BACKUPS IN {collection-location}`](show-backup.html#view-a-list-of-the-available-full-backup-subdirectories) will display a list of the full backup subdirectories in the collection's storage location.
+<!--TODO syntax to change here -->
+[`SHOW BACKUPS IN {collectionURI}`](show-backup.html#view-a-list-of-the-available-full-backup-subdirectories) will display a list of the full backup subdirectories at the collection's URI.
 
-Alternately, the following directories also constitute a backup collection. There are multiple backups in two separate storage locations. Each individual backup is a full backup and its related incremental backup(s). Despite using the [`incremental_location`](#incremental-backups-with-explicitly-specified-destinations) option to store the incremental backup in an alternative location, that incremental backup is still part of this collection as it depends on the full backup in the first cloud storage bucket:
+Alternately, the following directories also constitute a backup collection. There are multiple backups in two separate collection URIs. Each individual backup is a full backup and its related incremental backup(s). Despite using the [`incremental_location`](#incremental-backups-with-explicitly-specified-destinations) option to store the incremental backup in an alternative location, that incremental backup is still part of this backup collection as it depends on the full backup in the first cloud storage bucket:
```

Comment:
Perhaps, I think the current location is a good central reference point for this description... so I've added an anchor link in from the advanced section, in case a reader wants more information on how this works.